common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java

```diff
@@ -337,19 +336,12 @@ private void loadSecretsFromDb() throws IOException {
     FileSystem fs = FileSystem.getLocal(_conf);
     fs.mkdirs(new Path(secretsFile.getPath()), new FsPermission((short) 0700));
 
-    db = LevelDBProvider.initLevelDB(secretsFile, CURRENT_VERSION, mapper);
+    db = DBProvider.initDB(secretsFile, CURRENT_VERSION, mapper);
```

Comment:
https://github.com/apache/spark/blob/31ab8bc4d5df0c84cb5542200b286defea037cc2/common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java#L121-L124

When adding the `rocksdb` implementation, need to add new configuration to make the file name have `.rdb` suffix